lib/Chat/Parser/SystemMessage.php

```diff
@@ -509,6 +509,13 @@ protected function parseMessage(Message $chatMessage): void {
 				$parsedMessage = $this->l->t('{actor} shared a file which is no longer available');
 				if ($currentUserIsActor) {
 					$parsedMessage = $this->l->t('You shared a file which is no longer available');
+				} elseif ($currentActorType === Attendee::ACTOR_FEDERATED_USERS) {
+					$parsedMessage = $this->l->t('File shares are currently not supported in federated conversations');
```

Comment:
I'm curious, why message object is translated as normal message for conversations, so it contains file object in messageParameters e.t.c. Maybe we could get this information for messages as well?
LeftSidebar | conversationsStore | messagesStore
-- | -- | --
![image](https://github.com/nextcloud/spreed/assets/93392545/d14eccef-fd3a-4433-86bd-59b498f44ee3) | ![image](https://github.com/nextcloud/spreed/assets/93392545/b1690a45-0175-4a65-b231-95a68caa7846) | ![image](https://github.com/nextcloud/spreed/assets/93392545/8e4959d4-4cdd-4942-9de6-2df088b293c7)